segmentation_and_clustering.py

```diff
@@ -5,7 +5,7 @@
 from myploty import plot_point_cloud_plotly, plot_clusters_plotly
 
 
-def region_growing(point_cloud, normals, angle_threshold=91, distance_threshold=0.1):
+def region_growing(point_cloud, normals, angle_threshold=-91, distance_threshold=0.1):
```

Comment:
Useless function. Bad result started from here.
Update it with better structure instead of fixed angle threshold and distance threshold.